protocols/mdns/src/behaviour.rs

```diff
@@ -22,6 +22,16 @@ mod iface;
 mod socket;
 mod timer;
 
+use futures::{channel::mpsc, Stream, StreamExt};
+use if_watch::IfEvent;
+use libp2p_core::{transport::PortUse, Endpoint, Multiaddr};
+use libp2p_identity::PeerId;
+use libp2p_swarm::{
+    behaviour::FromSwarm, dummy, ConnectionDenied, ConnectionId, ListenAddresses, NetworkBehaviour,
+    THandler, THandlerInEvent, THandlerOutEvent, ToSwarm,
+};
+use smallvec::SmallVec;
```

Comment:
Please revert these formatting changes. We're using `group_imports =  "StdExternalCrate"` (you need to run `fmt` with nightly in order for it to be applied).

Reply:
understood 😅 apologies for the newbie mistakes

Reply:
No worries at all! :)